src/Spout/Reader/XLSX/Helper/SheetHelper.php

```diff
@@ -105,16 +108,18 @@ protected function getSheetFromSheetXMLNode($xmlReaderOnSheetNode, $sheetIndexZe
     {
         $sheetId = $xmlReaderOnSheetNode->getAttribute(self::XML_ATTRIBUTE_R_ID);
         $escapedSheetName = $xmlReaderOnSheetNode->getAttribute(self::XML_ATTRIBUTE_NAME);
+        $sheetState = $xmlReaderOnSheetNode->getAttribute(self::XML_ATTRIBUTE_STATE);
 
         /** @noinspection PhpUnnecessaryFullyQualifiedNameInspection */
         $escaper = \Box\Spout\Common\Escaper\XLSX::getInstance();
         $sheetName = $escaper->unescape($escapedSheetName);
+        $sheetVisible = ($sheetState === self::SHEET_STATUS_VISIBLE) ? true : false;
```

Comment:
you can remove the `? true : false` as this is redundant. `$sheetVisible = ($sheetState === self::SHEET_STATUS_VISIBLE);` is enough